### PR TITLE
Add zgadywanka game and script01

### PR DESCRIPTION
**Pull request by MartaSzymula**

### CODE REVIEW

File: zgadywanka.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import random
+import math
+minval=1
+maxval=10
+no=maxval-minval
```
Comment (WojciechKusa) [inline zgadywanka.py]: `no` is not the best variable name ;)

File: script_01.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import math
+import cmath
```
Comment (WojciechKusa) [inline script_01.py]: `cmath` is imported but not used, you can remove it

File: script_01.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+def task1(text="Count stuff in this random sentence."):
+    print(text)
+    vowels=['a','e','i','o','u','y']
```
Comment (WojciechKusa) [inline script_01.py]: please consider this document while reviewing your code with regard to whistespaces:
https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

File: script_01.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    consonants=list('bcdfghjklmnpqrstvwxyz')
+    sumofv=0
```
Comment (WojciechKusa) [inline script_01.py]: variable names should be more explanatory, e.q. `sum_of_vovels` instead of `sumofv`

File: script_01.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    for i in vowels:
+        print(str(i)+': '+str(text.count(i,0,-1)))
+        sumofv=sumofv+text.count(i,0,-1)
+    print('Sum of vowels:' +str(sumofv))
+    for a in consonants:
+        sumofc=sumofc+text.count(a,0,-1)
+    print('Sum of consonants:' + str(sumofc))
+    print('Number of words: ' + str(len(text.split(' '))))
+
+def task2(minval=1,maxval=100):
```
Comment (WojciechKusa) [inline script_01.py]: please add documentation to your functions and describe what are they doing. You can use this template:
```
def fun():
"""Function fun responsible for ..."""

```

File: script_01.py [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    even_numbers = []
+    for i in range(1,100):
+        if i%2==0:
+            even_numbers.append(i)
+    print (even_numbers)
```
Comment (WojciechKusa) [inline script_01.py]: there should be no whitespace between function name and its arguments

File: script_01.py [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+def task3(maxval=100):
+    uneven_no=[]
```
Comment (WojciechKusa) [inline script_01.py]: shouldn't it be `prime` instead of `uneven_no`?

**Review by WojciechKusa — COMMENTED** [review]
code is very good, please correct minor styling issues

File: script_01.py (hunk 3 above)
Comment (MartaSzymula) [reply script_01.py]: considered

**Review by WojciechKusa — APPROVED** [review]

**Pull request merged**